src/functional_data.Rmd

```diff
@@ -0,0 +1,177 @@
+# Functional Data
+
+Functional data provides information about curves varying over a continuum, such as time.
```

Comment:
Agree that there should be just a paragraph of explanation, especially on the how and why of FDA